### PR TITLE
Iterate on the circle flow

### PR DESCRIPTION
**Pull request by jcomo**
### Overview

Let's be honest. The original create circle flow was kinda...thangs. This new flow is legit. The circle creation is meant to have the same look and feel of the circle view. The fragment now has the ability to set the editing flag, which will display or hide guides (and edit functionality) as needed. This allows us to stick an edit button into the view circle fragment, although this hasn't been done yet.
### Technical changes
- Adds a rounded circle transform for picasso (with an option for a border)
- Allows users to select images to use for the circle profile avatar and background
- User can long click the name of the circle in edit mode to change the name
### Reviewers
- [x] @dbuscaglia 
- [ ] @hashamali

### CODE REVIEW

File: app/src/main/java/it/quip/android/actvitiy/LoginActivity.java [hunk 1]
```diff
@@ -53,7 +53,8 @@ public void onError(FacebookException exception) {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setupLoginButton();
+        // setupLoginButton();
+        startMainActivity();
```
Comment (hashamali) [inline app/src/main/java/it/quip/android/actvitiy/LoginActivity.java]: Undo them thangs here.

File: app/src/main/java/it/quip/android/fragment/CircleHeaderFragment.java [hunk 2]
```diff
@@ -1,22 +1,57 @@
...
+import it.quip.android.model.User;
+import it.quip.android.service.ImagePickerService;
+
+import static it.quip.android.util.StringUtils.isBlank;
 
 public class CircleHeaderFragment extends Fragment {
 
     private static final String CIRCLE = "it.quip.android.CIRCLE";
 
+    private static final int PICK_IMAGE_REQUEST = 1408;
```
Comment (hashamali) [inline app/src/main/java/it/quip/android/fragment/CircleHeaderFragment.java:32]: are these randomly chosen numbers?

File: app/src/main/java/it/quip/android/fragment/CircleHeaderFragment.java [hunk 3]
```diff
@@ -1,22 +1,57 @@
...
+    private static final int TAKE_CAMERA_IMAGE_REQUEST = 1556;
+
+    private static final int BACKGROUND_IMAGE = 1239;
+    private static final int AVATAR_IMAGE = 2871;
+
+    private int imageBeingEdited;
```
Comment (hashamali) [inline app/src/main/java/it/quip/android/fragment/CircleHeaderFragment.java]: gotta hit these with the 'm'

**Comment by dbuscaglia** [comment]
LGTM

**Comment by dbuscaglia** [comment]
LGTM

**Pull request merged**